history/cultures/sayaadi.txt

```diff
@@ -72,6 +71,7 @@
 	#
 	discover_innovation = innovation_rightful_ownership 
 	discover_innovation = innovation_cranes 
-	discover_innovation = innovation_primogeniture 
+	discover_innovation = innovation_primogeniture
 	#
+	discover_innovation = innovation_taiga_warfare
```

Comment:
Huh? What's the idea behind it?

Reply:
![image](https://user-images.githubusercontent.com/74790101/113480452-44635b00-9462-11eb-81d7-fbc598ef37e5.png)

605 start they are in hyjall in a taiga region